bigbluebutton-html5/imports/ui/components/presentation/presentation-menu/component.jsx

```
@@ -512,6 +531,17 @@ const PresentationMenu = (props) => {
       );
     }
 
+    if (props.amIPresenter && allowPopupPresentation) {
+      menuItems.push({
+        key: 'list-item-detach-presentation',
+        //label: intl.formatMessage({ id: 'app.presentation.detachPresentation' }),
+        label: formattedDetachedLabel(isPresentationDetached),
+        icon: 'external-link',
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)